.storybook/mixins/apollo-story-mixin.js

```diff
@@ -5,13 +5,13 @@ export default ({
 	loading = false,
 } = {}) => {
 	// Add to basket button expects basketAddInterstitial
-	const queryData = { ...queryResult, data: { ...queryResult.data, basketAddInterstitial: {} } };
+	const queryData = { ...queryResult.data, data: { ...queryResult.data, basketAddInterstitial: {} } };
```

Comment:
This might affect some other stories...there seem to be several that try to pull their data off of `data`...

Reply:
Are those instances using a `query` or `readQuery`? Queries pull from `data` and `readQuery` doesn't have `data` in the response.

Reply:
I'm wondering if `readQuery` has never working in our stories...

Reply:
For sure, will do, thanks.

Reply:
Clicked through the stories. Added some quick resolutions with missing mixins. There are some failing stories, but those seem to be from different/specific issues.

Reply:
And those failures exist in the deployed storybook instance as well.

Reply:
COol thank you!